systemtest/scripts/get_failures.sh

```diff
@@ -2,14 +2,14 @@
 
 JSON_FILE=failures.json
 
-FAILURES=$(find . -name 'TEST*.xml' -type f -print0 | xargs -0 grep "<testcase.*time=\"[0-9]\{1,\}\..*[^\/]>$" | cut -d '"'  -f 2,4)
+FAILURES=$(find . -name 'TEST*.xml' -type f -print0 | xargs -0 grep "<testcase.*time=\"[0-9]*,\{0,1\}[0-9]\{1,3\}\..*[^\/]>$" | cut -d '"'  -f 2,4)
```

Comment:
Just FYI `grep -E` means you don't have to `\`-quote metacharacters like `{` and `}`. 

Also, in general, `sed 's/.../.../` would be a slightly more robust way to do this, since you don't actually know that what you're `cut`ting is what you `grep`ped for. If your grep used `^` metacharacter then you'd know, but `sed` woudl still be more readable imho.